licenses/views.py

```diff
@@ -1,21 +1,21 @@
 from . import forms
 from .models import LicenseRequest
 from django.contrib.auth.decorators import login_required
-from django.shortcuts import render
 from django.urls import reverse_lazy
 from django.utils.decorators import method_decorator
 from django.views import generic
 
 
 @method_decorator(login_required, name='dispatch')
-class ListLicensesView(generic.View):
+class ListLicensesView(generic.list.ListView):
     """List all licenses of the user."""
 
     template_name = 'licenses/list.html'
+    model = LicenseRequest
 
-    def get(self, request):
-        """Handle get requests."""
-        return render(request, self.template_name, {})
+    def get_queryset(self):
+        """List only the licenses of the logged in user."""
+        return self.model.objects.filter(okuser=self.request.user)
```

Comment:
Hm, without super call we miss out the ordering feature from `ListView`. We do not need that here at the moment. We can also not specify a `queryset` on the class as we do not know the user beforehand. So let's keep it that way.